### PR TITLE
fix migrations

### PR DESCRIPTION
**Pull request by Chive**
Migration 0002 was not included in the git repo but in the package uploaded to Aldryn. Also, the changes
made in models.py in migration 0002 were never committed to the repo.

Therefore noop'ing the migration 0002 and reverting eventual changes back in 0003

### CODE REVIEW

File: aldryn_facebook/migrations/0002_auto__chg_field_facebookrecommendationbarplugin_site.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+    """ this migration file was not committed and therefore not included in
```
Comment (GaretJax) [inline aldryn_facebook/migrations/0002_auto__chg_field_facebookrecommendationbarplugin_site.py:9]: Horrible :D

**Comment by GaretJax** [comment]
LGTM

**Comment by stefanfoulis** [comment]
:+1:

**Comment by Chive** [comment]
Verified on the project were having issues on, success :)

**Pull request merged**